### ISSUE TITLE
🔥Dancing Dervish/Duo NOT WORK!

### ISSUE THREAD

**Issue by thisVioletHydra** [PathOfBuildingCommunity/PathOfBuilding#4076]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### What is the expected behaviour/value?

![image](https://user-images.githubusercontent.com/11839238/152610039-3b358d5d-6722-49aa-9366-f2a08216e1ed.png)

### What is the actual behaviour/value?

![image](https://user-images.githubusercontent.com/11839238/152610091-afdc682d-7331-43af-acb1-f4df7a6e27dc.png)
![image](https://user-images.githubusercontent.com/11839238/152610117-ac48c84d-69e3-47f4-a807-3a9d5066f183.png)
![image](https://user-images.githubusercontent.com/11839238/152610061-98411d3d-a0b7-41e5-87fa-56a95e7f6372.png)

### How to reproduce the issue

add `The Dancing Duo Reaver Sword`
```q
Rarity: UNIQUE
The Dancing Duo
Reaver Sword
Variant: Pre 3.11.0
Variant: Pre 3.17.0
Selected Variant: 2
Reaver Sword
Quality: 20
Sockets: G-G-G-G-G-G
LevelReq: 65
Implicits: 2
{variant:1}40% increased Global Accuracy Rating
{variant:2}60% increased Global Accuracy Rating
{variant:1}{range:0.5}(160-190)% increased Physical Damage
{variant:2}{range:0.5}(130-160)% increased Physical Damage
{range:0.5}(25-30)% increased Attack Speed
5% increased Movement Speed
Rampage
Triggers Level 15 Manifest Dancing Dervishes on Rampage
Manifested Dancing Dervish disables both weapon slots
Manifested Dancing Dervish dies when Rampage ends
```

### Build code

```shell
https://pastebin.com/0vePV94g
```

**Comment by MartinAntoniuk** [comment]
+1 Happens with both the now legacy The Dancing Duo and the now only sword The Dancing Dervish.

I can't see any important stat in my build.

**Comment by Vara88** [comment]
+1 i can't see the stats in my build, too.
Hope this gets taken care of soon.

**Comment by MosheWork** [comment]
same hope they fix it im flying blind here :)

**Comment by Azhkanizkael** [comment]
+1 unable can't see anything for Manifest Dancing Dervish

**Comment by Wires77** [comment]
Fixed in v2.15.0
